schedule/yam/agama_raid0_unattended.yaml

```diff
@@ -0,0 +1,12 @@
+---
+name: agama_raid0_unattended
```

Comment:
just `raid` as it will be reused for other raid levels.

Reply:
updated, thanks.